Directory.Build.targets

```diff
@@ -12,6 +12,16 @@
     <DefaultExcludesInProjectFolder Condition="'$(TargetFrameworkIdentifier)' != '.NETFramework'">$(DefaultExcludesInProjectFolder);**/*.netframework$(DefaultLanguageSourceExtension)</DefaultExcludesInProjectFolder>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <IsSourceProject Condition="'$(IsSourceProject)' == '' and
+                                '$(IsTestProject)' != 'true' and
+                                '$(UsingMicrosoftTraversalSdk)' != 'true'">true</IsSourceProject>
+    <CLSCompliant Condition="'$(CLSCompliant)' == '' and
+                             ('$(IsTestProject)' == 'true' or
+                              '$(ExcludeFromAssemblyMetadata)' == 'true')">false</CLSCompliant>
```

Comment:
Setting the CLSCompliant property to false isn't needed as it's already opt-in via the line below (22):

```suggestion
```

Reply:
So we don't care if it's not explicitly indicated even if it's set to false?

Reply:
Right. We only care if it's true.